front/src/common/Map/Buttons/MapButtons.tsx

```diff
@@ -55,6 +56,7 @@ export default function MapButtons({
   bearing,
   editorProps,
   viewPort: viewportProps,
+  isNewButtons,
```

Comment:
Set it to false by default, this way you can remove the props in the components which do not need the new buttons
```suggestion
  isNewButtons = false,
```

Reply:
Done